hcache.c

```diff
@@ -740,12 +735,11 @@ header_cache_t *
 mutt_hcache_open(const char *path, const char *folder, hcache_namer_t namer)
 {
   const hcache_ops_t *ops = hcache_get_ops();
-  header_cache_t *h = safe_calloc(1, sizeof (header_cache_t));
-  struct stat sb;
-
   if (!ops)
     return NULL;
 
+  header_cache_t *h = safe_calloc(1, sizeof (header_cache_t));
+
   /* Calculate the current hcache version from dynamic configuration */
   if (hcachever == 0x0) {
```

Comment:
Aren't you missing a `struct stat sb;`? From the code it seems it's not used in the function, but since you deleted, I'd rather make sure. Other than that, LGTM!